code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -52,7 +52,12 @@
 	. = ..()
 	if(!HAS_TRAIT(src, TRAIT_AUTO_CATCH_ITEM) && !skip_throw_mode_check && !in_throw_mode)
 		return
-	if(get_active_held_item() || restrained())
+	if(restrained())
+		return
+	if (get_active_held_item())
+		if (HAS_TRAIT_FROM(src, TRAIT_AUTO_CATCH_ITEM,RISING_BASS_TRAIT))
+			visible_message("<span class='warning'>[src] chops [I] out of the air!</span>")
+			return TRUE
```

Comment:
FYI, this "break" was intended because stopping/deflecting every thrown item regardless of  bare hand or not is pretty damn stupid.
Whatever replace the `restrained` proc check above with an `incapacitated` one.

Reply:
yes because stopping bolas and tile throwing wasn't what I had in mind when I coded that specifically